packages/devtools-reps/src/reps/tests/string-with-url.js

```diff
@@ -16,6 +16,21 @@ const renderRep = (string, props) =>
     })
   );
 
+const simulateClick = (link, openLink, url) => {
```

Comment:
it would be nice to call it "testLinkClick" or something similar since we do the click and also run some assertions related to that click.